prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java

```diff
@@ -277,6 +280,15 @@ public void testRemoveEmptyChannel() {
                 CliConstants.DIR, dir.toString());
         Assert.assertEquals(ReturnCodes.INVALID_ARGUMENTS, exitCode);
     }
+    @Test
```

Comment:
Other tests:
 * error when `--channel` is used together with `--manifest` or `--repositories`
 * error when `--channel` is used with non-existing file
 * error when `--channel` is used with invalid channel file
 * verify that the recorded channel has the same content (minus name) as the one passed in